### ISSUE TITLE
Some requirements found on twitter

### ISSUE THREAD

**Issue by mgmart** [MobileOrg/mobileorg.next#5]
4/ I want Auto Layout. Notifications. Background sync. 3D Touch. Sharing to/from. Better Dropbox support. Latest keyboard support. Etc.

https://twitter.com/0xcmp/status/735705998917132288

**Comment by mgmart** [comment]
3D touch has to be incorporated into specification

Anyone an idea what 

> latest keyboard support

could mean?

**Comment by abradd** [comment]
Perhaps he is referring to custom keyboards?

Not sure what the purpose of 3D touch is in this instance.. maybe peeking into tasks?

**Comment by webframp** [comment]
3D touch or force touch on app icon could allow quick capture, or daily agenda summary

**Comment by chrisportela** [comment]
I'll provide clarification as author of the Tweet. Latest keyboard support is just not needing to use compatibility mode. I have a 6s Plus so being able to use the new keyboard + ForceTouch scrolling is part of my everyday. I believe simply updating the API version fixes that.

3D touch would be mainly for actions to do things or to quickly get to a specific area inside the app like @webframp said.

**Comment by mgmart** [comment]
> I have a 6s Plus so being able to use the new keyboard + ForceTouch scrolling is part of my everyday. I believe simply updating the API version fixes that.

@0xCMP, is that requirement fulfilled by [v1.7.1](https://itunes.apple.com/gb/app/mobileorg/id634225528?mt=8)?

**Comment by chrisportela** [comment]
Yes, @mgmart! didn't notice it had been updated. That's a major improvement on it's own for me.
